src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/EditApplicationHandler.java

```diff
@@ -120,7 +120,8 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<BailCase
         requireNonNull(callback, "callback must not be null");
```

Comment:
Can we please rename the handler name, as it will be used to remove fields after certain event, which can be other than EditApplication too.